orion/handlers/http/http.go

```diff
@@ -103,7 +104,13 @@ func prepareContext(req *http.Request, info *methodInfo) context.Context {
 	// fetch and populate whitelisted headers
 	if len(info.svc.requestHeaders) > 0 {
 		for _, hdr := range info.svc.requestHeaders {
-			ctx = headers.AddToRequestHeaders(ctx, hdr, req.Header.Get(hdr))
+			if values, found := req.Header[textproto.CanonicalMIMEHeaderKey(hdr)]; found {
```

Comment:
req.Header.Get involves CanonicalMIMEHeaderKey so we use the same format accessing headers map